Offseason2022/src/main/java/frc/robot/commands/DriveQuickturn.java

```diff
@@ -14,12 +15,16 @@ public DriveQuickturn( )
   {
     // m_subsystem = subsystem;
     // addRequirements(m_subsystem);
+    setName("DriveQuickturn");
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize( )
-  {}
+  {
+    RobotContainer robotContainer = RobotContainer.getInstance( );

```

Comment:
These two lines can be handled by
RobotContainer.getInstance().m_drivetrain.moveSetQuickTurn(true);

Don't need to have a separate line for getInstance()